utilities/src/main/java/org/craftercms/commons/config/ConfigUtils.java

```diff
@@ -86,6 +106,9 @@ public static HierarchicalConfiguration<ImmutableNode> readXmlConfiguration(Inpu
             if (MapUtils.isNotEmpty(prefixLookups)) {
                 xmlParams = xmlParams.setPrefixLookups(prefixLookups);
             }
+            if (isNotEmpty(lookupVariables)) {
+                xmlParams = xmlParams.setDefaultLookups(getLookups(lookupVariables));
```

Comment:
Wouldn't this break up existing default lookups?

Reply:
There should be no other lookups, we are creating a brand new `XMLBuilderParameters`. Other prefix lookups will still work

Reply:
What about lookups like `env`?